roles/edpm_ovn_bgp_agent/templates/ovn_bgp_agent.yaml.j2

```diff
@@ -0,0 +1,24 @@
+start_order: 0
+image: "{{ edpm_ovn_bgp_agent_image }}"
+net: host
+pid: host
+cgroupns: host
+restart: always
+privileged: true
+healthcheck:
```

Comment:
We no longer install healtcheck script in container images. If we still need it then we have to discuss how we install this script. TBH healthcheck scripts have not been well-maintained so we can just drop all healhchecks.

Reply:
ok, so just removing those 2 lines, right?

Reply:
Yes. That's correct